contracts/ContractRecovery.sol

```diff
@@ -97,9 +97,9 @@ contract ContractRecovery is CommonStorage {
 
   // Can only be called by the root role.
   function removeRecoveryRole(address _user) public stoppable auth {
-    if (CommonAuthority(address(authority)).hasUserRole(_user, RECOVERY_ROLE)) {
-      CommonAuthority(address(authority)).setUserRole(_user, RECOVERY_ROLE, false);
-      recoveryRolesCount--;
+    if (CommonAuthority(address(authority)).hasUserRole(_user, RECOVERY_ROLE)) { // ignore-swc-113 ignore-swc-128
```

Comment:
How is this a SWC 128 violation? I'm pretty sure this is a constant-time lookup.

Reply:
It's a false positive I haven't logged since it's not present in v 1.5.0 onwards. See this build for example which is running v1.5.1 https://circleci.com/gh/JoinColony/colonyNetwork/8796